src/HotChocolate/Spatial/test/Types.Tests/__snapshots__/GeoJsonMultiLineStringInputTests.Schema_Tests.snap

```diff
@@ -3,29 +3,32 @@
 }
 
 type Query {
-  test(arg: MultiPolygonInput): String
+  test(arg: GeoJSONMultiLineStringInput): String
 }
 
-input MultiPolygonInput {
+input GeoJSONMultiLineStringInput {
+  "The geometry type of the GeoJson object"
+  type: GeoJSONGeometryType
+  "The \"coordinates\" field is an array of LineString coordinate arrays."
   coordinates: [[Position]]
+  "The coordinate reference system integer identifier"
   crs: Int
-  type: GeoJSONGeometryType
 }
 
 enum GeoJSONGeometryType {
-  POINT
-  MULTIPOINT
-  LINESTRING
-  MULTILINESTRING
-  POLYGON
-  MULTIPOLYGON
-  GEOMETRYCOLLECTION
+  Point
+  MultiPoint
+  LineString
```

Comment:
this does not follow the graphql spec recommendations.... why did we choose to not follow it?

Reply:
because it is geojson spec. this way you can just pass objects as variables